src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -79,6 +99,22 @@ public CaseSearchResult searchCases(
         return result;
     }
 
+    private List<String> getCaseTypeIds(List<String> caseTypeIds) {
+        if (isAConsolidationQuery(caseTypeIds)) {
+            return getCaseTypes();
+        }
+        return caseTypeIds;
+    }
+
+    private List<String> getCaseTypes() {
+        final List<String> roles = userService.getUserRolesJurisdictions();
+        return caseDefinitionRepository.getAllCaseTypesByJurisdictions(roles).get();
+    }
+
+    private boolean isAConsolidationQuery(List<String> caseTypeIds) {
+        return ElasticsearchRequest.SOURCE_WILDCARD.equals(caseTypeIds.get(0));
```

Comment:
```suggestion
        return ElasticsearchRequest.ANY_CASE_TYPE.equals(caseTypeIds.get(0));
```

Reply:
I think some other name would be better - maybe just `WILDCARD`? The constant was already being used as a possible value that can be used inside `_source`, so it now has two completely separate uses.

Reply:
ok for me

Reply:
DONE